app/scripts/components/modelcard.js

```diff
@@ -3,17 +3,43 @@ var exports = (function () {
 
   var ModelCard = Vue.component("model-card", {
     template: "#template-model-card",
+
+    data: function() {
+      return {
+        selected: false
+      };
+    },
+
     props: {
       model: {
         required: true
+      },
+      selectable: {
+        required: false
       }
     },
+
     methods: {
-      toggleActive: function(model) {
-        // we can only toggle on the parent, because we have to clear other actives
-        this.$parent.toggleActive(model);
+      toggleActive: function() {
+        this.model.active = !this.model.active;
+        this.$dispatch("deactivateall", this.model);
```

Comment:
dispatch is removed in vue 2.0, in favour of properties or bus communication. (http://vuejs.org/guide/migration.html#dispatch-and-broadcast-deprecated). Event naming is inconsistent deactivateall vs select-all